setup.py

```diff
@@ -54,7 +54,7 @@ def _get_test_env():
 
     def run(self):
 
-        cmd = ['py.test']
+        cmd = ['py.test -vv']
```

Comment:
In that case is it possible to potentially modify this ```setup.cfg``` on the Jenkins machine at all?
Otherwise we're going to keep putting this in/taking it out and we'll likely end up missing it once and it'll go in. Or we have to keep checking for this which is making work for work's sake.